packages/autocomplete-core/src/resolve.ts

```diff
@@ -56,15 +60,36 @@ type RequestDescriptionPreResolvedCustom<TItem extends BaseItem> = {
 
 export function preResolve<TItem extends BaseItem>(
   itemsOrDescription: TItem[] | TItem[][] | RequesterDescription<TItem>,
-  sourceId: string
+  sourceId: string,
+  state: AutocompleteState<TItem>
 ):
   | RequestDescriptionPreResolved<TItem>
   | RequestDescriptionPreResolvedCustom<TItem> {
   if (isRequesterDescription<TItem>(itemsOrDescription)) {
+    const contextParameters =
+      itemsOrDescription.requesterId === 'algolia'
```

Comment:
is this branch required (I think so)?

Reply:
This identifies Algolia requesters, so yes.